apps/web/src/components/Onboarding/VerificationFlow/useIsAccountVerified.tsx

```diff
@@ -3,5 +3,5 @@ import { useCurrentAccount } from "@umami/state";
 export const useIsAccountVerified = () => {
   const account = useCurrentAccount();
 
-  return account?.type !== "mnemonic" || account.isVerified;
+  return !!account && (account.type !== "mnemonic" || account.isVerified);
```

Comment:
is there a difference? also you don't need the double negation operator anyway

Reply:
In the previous implementation, `useIsAccountVerified` gave true if the account was not defined